packages/examples/basic/package.json

```diff
@@ -2,9 +2,11 @@
   "private": "true",
   "scripts": {
     "start": "DISABLE_ESLINT_PLUGIN=true react-scripts start",
-    "analyze": "source-map-explorer 'build/static/js/*.js' --gzip",
+    "build": "yarn build && yarn build:app",
```

Comment:
do you mean yarn build:blade? also do you need this in build of this app? this command only needs build app right? build:blade is required just for analyse right?

Reply:
So think of this scenario-
- You add new component to blade.
- Now you have to build blade to be able to use that component inside this example.
- Then you can write the new code in example and build the app

build:app without build:blade will use the older version of blade. So the build:app may or may not work without build:blade.